recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domu-vdevices.cfg

```diff
@@ -0,0 +1,20 @@
+# =====================================================================
+# DomU virtual devices
+# =====================================================================
+
```

Comment:
There was a comment here "# Uncomment this when using block device as a boot device" before moving

Reply:
I remove it intentionally because as I see line after "Uncomment" is already uncommented. If you think it's important, I will rephrase it to something like "if .. comment else .. uncomment"

Reply:
Whether "disk = ..." string should be (un)commented really depends on "extra = ... " string above. 

If nfs is used as a boot device then "disk = ..." should be commented, but if block device (emmc, sd, whatever) is used as a boot device then "disk = ..." should be **un**commented.

Reply:
Done.